react/components/molecules/date-input/date-input.js

```diff
@@ -0,0 +1,162 @@
+import React, { PureComponent } from "react";
+import { Platform, StyleSheet, Text, ViewPropTypes, View } from "react-native";
+import DateTimePicker from "@react-native-community/datetimepicker";
+import { mix } from "yonius";
+import PropTypes from "prop-types";
+
+import { IdentifiableMixin, baseStyles } from "../../../util";
+
+import { ContainerOpenable, Icon, Touchable } from "../../atoms";
+
+export class DateInput extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            value: PropTypes.instanceOf(Date),
+            header: PropTypes.string,
+            disabled: PropTypes.bool,
+            activeOpacity: PropTypes.number,
+            onUpdateValue: PropTypes.func,
+            style: ViewPropTypes.style
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            value: new Date(),
+            header: undefined,
+            disabled: false,
+            activeOpacity: 0.75,
+            onUpdateValue: () => {},
+            style: {}
+        };
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            valueData: this.props.value,
+            visible: false
+        };
+    }
+
+    onPress = () => {
+        this.setState({
+            visible: true
+        });
+        if (Platform.OS === "ios") this.container.toggle();

```

Comment:
Shouldn't be possible but better to be safe than sorry
```suggestion
        if (Platform.OS === "ios" && this.container) this.container.toggle();

```